assets/rnadeseq_report.Rmd

```diff
@@ -2823,32 +3095,34 @@ if (software_versions_csv) {
 
-```{r no_software_versions, echo=FALSE, results='asis', eval=!isProvided(params$path_software_versions)}
-cat("No software versions file was provided for primary analysis.")
-```
+
 
 ```{r software_versions_RNASeq, echo=FALSE, results='asis', eval=isProvided(params$path_software_versions)}
 
-# In the following part, I use newlines (in code) where possible whenever the paste0 function switches from a string to a variable call to better distinguish between the two so it is clearer in which lines quotes are needed and in which not --> Cannot always use newlines as this sometimes translates into the report.html
+# In the following part, I use newlines (in code) where possible whenever the paste0 function switches from a string to a variable call to
+# better distinguish between the two so it is clearer in which lines quotes are needed and in which not
+# --> Cannot always use newlines as this sometimes translates into the report.html
 
-if (params$input_type == "featurecounts"){
+if (params$input_type == "featurecounts" || params$input_type == "salmon" || params$input_type == "rsem"){
 cat(paste0(
-"## RNAseq data analysis
+"## Primary Bioinformatics
 
-The Nextflow-based nf-core pipeline `rnaseq ", as.character(ifelse(software_versions_csv, version["nf-core/rnaseq"], version["Workflow"][[1]]["nf-core/rnaseq"])),
-"` [^1] was used for the RNAseq Bioinfomatics analysis. `FASTQC ",
-as.character(ifelse(software_versions_csv, version["FastQC"], version["FASTQC"][[1]]["fastqc"])),
-"` [^3] [@andrews2010fastqc] was used to determine quality of the FASTQ files.
-Subsequently, adapter trimming was conducted with `Trim Galore ",
+### RNASeq analysis {#rnaseq-methods}
+
+The Nextflow-based [nf-core/rnaseq](https://nf-co.re/rnaseq) pipeline, revision ", as.character(ifelse(software_versions_csv, version["nf-core/rnaseq"],
+version["Workflow"][[1]]["nf-core/rnaseq"])),
+", was used for the [primary bioinformatic analysis](#primary-bioinformatics-rnaseq).
+[FASTQC ", as.character(ifelse(software_versions_csv, version["FastQC"], version["FASTQC"][[1]]["fastqc"])),
+"](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) [@andrews2010fastqc] was used to determine the quality of the FASTQ files.
+Subsequently, adapter trimming was conducted with [Trim Galore ",
 as.character(ifelse(software_versions_csv, version["Trim Galore!"], version["TRIMGALORE"][[1]]["trimgalore"])),
-"` [^4] [@krueger2012trim]. `STAR v",
-as.character(ifelse(software_versions_csv, substring((version["STAR"]), 7, ), version["MAKE_TRANSCRIPTS_FASTA"][[1]]["star"])),
-"` [@Dobin2013] aligner was used to map the reads that passed the quality control to the reference genome.
-The RNA-seq data quality control was performed with `RSeQC ",
+"](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/) [@krueger2012trim].
+The [STAR v", as.character(ifelse(software_versions_csv, substring((version["STAR"]), 7, ), version["MAKE_TRANSCRIPTS_FASTA"][[1]]["star"])),
```

Comment:
Here starts a mixture of `v<version>` or `version <version>` or even only `<version>`, uniformity would be good, I actually prefer `v<version>`.